src/EventService.Business/Commands/EventUser/CreateEventUserCommand.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using FluentValidation.Results;
+using LT.DigitalOffice.EventService.Broker.Requests.Interfaces;
+using LT.DigitalOffice.EventService.Business.Commands.EventUser.Interfaces;
+using LT.DigitalOffice.EventService.Data.Interfaces;
+using LT.DigitalOffice.EventService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.EventService.Models.Db;
+using LT.DigitalOffice.EventService.Models.Dto.Enums;
+using LT.DigitalOffice.EventService.Models.Dto.Requests.EventUser;
+using LT.DigitalOffice.EventService.Validation.EventUser.Interfaces;
+using LT.DigitalOffice.Kernel.BrokerSupport.AccessValidatorEngine.Interfaces;
+using LT.DigitalOffice.Kernel.Constants;
+using LT.DigitalOffice.Kernel.Extensions;
+using LT.DigitalOffice.Kernel.Helpers.Interfaces;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.Models.Broker.Models;
+using Microsoft.AspNetCore.Http;
+
+namespace LT.DigitalOffice.EventService.Business.Commands.EventUser;
+
+public class CreateEventUserCommand : ICreateEventUserCommand
+{
+  private readonly IAccessValidator _accessValidator;
+  private readonly IEventUserRepository _repository;
+  private readonly IDbEventUserMapper _mapper;
+  private readonly ICreateEventUserRequestValidator _validator;
+  private readonly IResponseCreator _responseCreator;
+  private readonly IHttpContextAccessor _contextAccessor;
+  private readonly IEventRepository _eventRepository;
+  private readonly IEmailService _emailService;
+  private readonly IUserService _userService;
+  public CreateEventUserCommand(
+    IAccessValidator accessValidator,
+    IEventUserRepository repository,
+    IDbEventUserMapper mapper,
+    ICreateEventUserRequestValidator validator,
+    IResponseCreator responseCreator,
+    IHttpContextAccessor contextAccessor,
+    IEventRepository eventRepository,
+    IEmailService emailService,
+    IUserService userService)
+  {
+    _accessValidator = accessValidator;
+    _repository = repository;
+    _mapper = mapper;
+    _validator = validator;
+    _responseCreator = responseCreator;
+    _contextAccessor = contextAccessor;
+    _eventRepository = eventRepository;
+    _emailService = emailService;
+    _userService = userService;
+  }
+
+  private async Task SendInviteEmailsAsync(List<Guid> userIds, string eventName)
```

Comment:
That's pretty, but Marina said in the project it is customary to do otherwise..
* fields
* private
* ctor
* public